extensions/amp-consent/amp-consent.md

```diff
@@ -202,7 +202,7 @@ The consent decisions collected from user via this prompt UI will be stored in `
 
 `geoOverride` provides a way to utilize the `<amp-geo>` component to detect user's geo location to assist client side decisions.
 
-`geoOverride` is a JSON object keyed by geo group codes which are defined in `<amp-geo>` (details [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-geo/amp-geo.md)). Each geo override should be a valid `<amp-consent>` config object. AMP will take all the values in the corresponding `geoOverride` to override the existing config.
+`geoOverride` is a JSON object keyed by geo group codes which are defined in `<amp-geo>` (details [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-geo/amp-geo.md)). Each geo override should be a valid `<amp-consent>` config object. AMP will take all the values in the corresponding `geoOverride` to override the existing config. The only exception here is that the `consentInstanceId` cannot be overwritten by `geoOverride` config, because AMP only manage and store consent information in a single instance.
```

Comment:
Should we mention CMP, policy objects being ok to be able to be merged?

Reply:
Should be fine? Since we had  `AMP will take all the values in the corresponding `geoOverride` to override the existing config` already. 

I'm leaning towards being less flexible at this moment, but fine with this in general. Is there another variable that should never be overwritten that you can think of?

Reply:
Merged the doc. Happy to change it, let me know. Thanks